TACC/Core-Styles
# feat: do not uppercase tags

## Overview

Do not force uppercase on tag text.

## Related

- [TUP-584](https://tacc-main.atlassian.net/browse/TUP-584)

## Changes

- **removed** forced uppercase on tags
- **changed** example tag text

## Testing

1. Open http://localhost:3000/components/detail/c-tag--types.
2. Verify tag text is **not** all uppercase.

## UI

| before | after |
| - | - |
| ![before](https://github.com/TACC/Core-Styles/assets/62723358/eb5afc7c-4664-4abd-a151-126717311c16) | ![after](https://github.com/TACC/Core-Styles/assets/62723358/cc8a013c-94de-49f1-866e-de960d1011aa) |
